src/corehost/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required (VERSION 2.6)
 
+project(corehost)
```

Comment:
This doesn't seem to be related to the issue at hand. But I'm not a CMAKE expert, so I might be wrong.

Reply:
I see - thanks for the explanation. OK - it seems we missed this for a while now.